src/login.html

```diff
@@ -224,7 +226,8 @@ <h1 class="text-primary mb-4 title" id="createAcc">Create new account</h1>
                                     <input class="form-control" id="register-password-control" type="password"
                                         placeholder="Password" required>
                                     <small class="invalid-feedback" role="alert"></small>
-                                    <small id="register-password-help" class="form-text text-muted" role="alert">Your password must
+                                    <small id="register-password-help" class="form-text text-muted" role="alert">Your
```

Comment:
Maybe removing the role would fix the login page.

Reply:
I think the role is needed.. what do you say @clivecharles